packages/apps-engine/src/server/AppManager.ts

```diff
@@ -742,7 +743,7 @@ export class AppManager {
      * With an instance of a ProxiedApp, start it up and replace
      * the reference in the local app collection
      */
-    public async updateLocal(stored: IAppStorageItem, appPackageOrInstance: ProxiedApp | Buffer) {
+    public async updateLocal(stored: IAppStorageItem, appPackageOrInstance: ProxiedApp | Buffer, enable: boolean) {
```

Comment:
Instead of introducing a boolean parameter, what about creating two methods? https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/